lib/galaxy/tool_util/verify/script.py

```diff
@@ -339,8 +342,10 @@ def _arg_parser():
     parser.add_argument('--verbose', default=False, action="store_true", help="Verbose logging.")
     parser.add_argument('-c', '--client-test-config', default=None, help="Test config YAML to help with client testing")
     parser.add_argument('--suite-name', default=DEFAULT_SUITE_NAME, help="Suite name for tool test output")
-    parser.add_argument('--skip-with-reference-data', default=False, action="store_true", help="Skip tests the Galaxy server believes use data tables or loc files.")
-    parser.add_argument('--history-per-test-case', default=False, action="store_true", help="Create new history per test case.")
+    parser.add_argument('--with-reference-data', dest="with_reference_data", default=False, action="store_true")
```

Comment:
I think you need to adjust the parameters to use with_reference_data now.

Reply:
I think I did this now?